examples/template-override.yaml

```diff
@@ -8,33 +8,41 @@ metadata:
   name: guestbook
 spec:
   generators:
-  - git:
-      repoURL: https://github.com/infra-team/cluster-deployments.git
-      files:
-      - path: "**/kustomization.yaml"
+  - list:
```

Comment:
It looks like you're replacing the Git files generator example with a list generator example, but I assume the original example is still valid? Another option could be a separate `template-overrides-list.yaml` that contains the list changes.

Reply:
The Git example is harder to test locally that's why I changed it. I'm also working on a git example that will work. 
Do you think we can merge it in the meantime?

Reply:
Sure, sounds good.